### ISSUE TITLE
Running `prisma generate` calculates a new `#mermaid-` id each time in the generated SVG

### ISSUE THREAD

**Issue by heystevegray** [keonik/prisma-erd-generator#3]
All of the `#mermaid-1626460027650` strings look like they are using a Unix timestamp. When you run `prisma generate`, the output file's `#mermaid-` id's are updated and it appears as a file change in git. See the diff image below:

![image](https://user-images.githubusercontent.com/66500112/125993870-07de7559-7894-401a-869e-c43ccc766289.png)

**Comment by heystevegray** [comment]
@keonik Looks like you can override the [deterministicIds](https://mermaid-js.github.io/mermaid/#/./Setup?id=deterministicids) setting in the mermaid configuration.

**Comment by keonik** [comment]
Good idea. I was getting a little peeved it was updating in git for every commit.

**Comment by keonik** [comment]
@heystevegray After testing this in my projects the id's are still making their way through. It looks like the problem could be the path you provided to the config file. The config file also might need to be outside of `src` because when someone installs the package through npm they run from the `dist` directory

**Comment by keonik** [comment]
@heystevegray I had to comment this change out. It was causing issues and wasn't generating ER Diagrams anymore #9
